src/controller/CHIPDeviceController.cpp

```diff
@@ -76,6 +76,10 @@ constexpr const char kPairedDeviceListKeyPrefix[] = "ListPairedDevices";
 constexpr const char kPairedDeviceKeyPrefix[]     = "PairedDevice";
 constexpr const char kNextAvailableKeyID[]        = "StartKeyID";
 
+#if CHIP_DEVICE_CONFIG_ENABLE_MDNS
+constexpr uint16_t kMdnsPort = 5353;
```

Comment:
Can `StartResolver` take 5353 as default? I can't think of a reason why one would like to call it with a different value.

Reply:
on non-embedded we may want to use a different port to not conflict with the platform daemon likely already listening on 5353. In that case we probably could use another port and ask for unicast replies when performing queries.